product_docs/docs/pem/8/pem_query_tool.mdx

```diff
@@ -238,26 +238,26 @@ Use the `Connection status` feature to view the current connection and transacti
 
 ## Macros
 
-Query Tool Macros enable you to execute pre-defined SQL queries with a single key press. Pre-defined queries can contain the placeholder $SELECTION$. Upon macro execution, the placeholder will be replaced with the currently selected text in the Query Editor pane of the Query Tool.
+Query Tool Macros enable you to execute pre-defined SQL queries with a single key-press. Pre-defined queries can contain the placeholder $SELECTION$. Upon macro execution, the placeholder will be replaced with the currently selected text in the Query Editor pane of the Query Tool.
```

Comment:
No hyphen in predefined. "key-press" is jargon, so I would rewrite as shown. No future tense unless you're really talking about a future activity. Avoid -tion words. "Upon macro execution" was a complicated way of saying that anyhow...

Query Tool macros enable you to execute predefined SQL queries by pressing a single key. Predefined queries can contain the placeholder `$SELECTION$`. When the macro executes, the placeholder is replaced with the currently selected text in the **Query Editor** pane of the Query Tool.